packages/shared-pure/src/types/AmountConfigEntry.ts

```diff
@@ -54,3 +54,19 @@ export interface AmountConfigBase {
   category: Token['category']
   bridgedUsing?: Token['bridgedUsing']
 }
+
+export interface AggLayerL2Token extends AmountConfigBase {
```

Comment:
Please add this type to union on top of this file, update createAmountId.ts and its tests, resolve the type issues. This is not needed for now but will make next PRs shorter